### ISSUE TITLE
Fix the build to use dart-sass (unblock node upgrade) 

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/va.gov-team#72830]
Fix the build to use dart-sass (unblock node upgrade) Initiative

**Comment by humancompanion-usds** [comment]
This issue is blocked by Formation deprecation.

**Comment by humancompanion-usds** [comment]
@micahchiang has a [PR](https://github.com/department-of-veterans-affairs/vets-website/pull/31931) going that would update us to a specific version of dart-sass and unblock the node upgrade.

**Comment by humancompanion-usds** [comment]
Adrian forked a branch off of Micah's original branch. Adrian resolved the networking issue for Cypress. Curt noticed there was a unit test stress test silent failure. Failing on a node child process. He's going to dedicate time to that this week. But Curt is also in the path of  the hurricane. Targeting 11/11 to have this done.

**Comment by humancompanion-usds** [comment]
Curt and Clint talked recently. We're going to compile a list of tests and applications that need to be fixed then determine a strategy to roll out to teams. Might skip tests that are broken, depending on how many tests are broken. Ian, Ray, and Harshil are focused on platform tests. Fixing all of the tests is the long pole.

**Comment by humancompanion-usds** [comment]
@micahchiang and @caw310 - Do we need a sub-issue attached to this initiative to make the final switch to dart-sass? Or is that already done?
